src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarHeader.cs

```diff
@@ -16,12 +16,12 @@ namespace System.Formats.Tar
     internal partial struct TarHeader
     {
         // POSIX fields (shared by Ustar and PAX)
-        private const string UstarMagic = "ustar\0";
-        private const string UstarVersion = "00";
+        internal const string UstarMagic = "ustar\0";
+        internal const string UstarVersion = "00";
 
         // GNU-specific fields
-        private const string GnuMagic = "ustar ";
-        private const string GnuVersion = " \0";
+        internal const string GnuMagic = "ustar ";
+        internal const string GnuVersion = " \0";
```

Comment:
Need to revert this. I initially intended to write the magic and version in the conversion constructors, but it isn't necessary, since it is written into the header when the entry is written to an archive.